app/server/lib/OIDCConfig.ts

```diff
@@ -167,33 +224,43 @@ export class OIDCConfig {
         profile,
       }));
 
-      delete mreq.session.oidc;
+      mreq.session.oidc = {
```

Comment:
I'd appreciate one more comment here that we are intentionally dropping the temporary state used for auth, since the auth has just completed.